Sources/Commands/Options.swift

```diff
@@ -136,6 +136,12 @@ public struct SwiftToolOptions: ParsableArguments {
     @Option(help: "Specify build/cache directory")
     var buildPath: AbsolutePath?
 
+    @Option(help: "Specify the global repository cache directory")
+    var cachePath: AbsolutePath?
+
+    @Flag(help: "Skip the cache when fetching")
```

Comment:
Should this mention the repository cache specifically, or is this intended to skip all caches (including any ones added in the future)?